### ISSUE TITLE
Option to remove hotkey in Settings -> Keyboard Shortcuts

### ISSUE THREAD

**Issue by mainakchhari-kv** [warpdotdev/Warp#2300]
### Describe the solution you'd like?

```text
If there are conflicts in keybinds, user should be able to remove one of them instead of having to reassign compulsarily
```

### Is your feature request related to a problem? Please describe.

After the recent update (version v0.2022.11.29.08.03.stable_03), suddenly my reassigned shortcuts have stopped working. Closer look revealed a few keybind conflicts which were never there earlier (interestingly, not the ones that had stopped working). On clicking on any keybinds, user is presented with 3 options, Reset to defaults, Cancel and Save. User should be able to just deassign a keybind apart of the above 3.

### How important is this feature to you?

5 (Can't work without it!)

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Hi @mainakchhari-kv Thanks for this feature request! Please :+1:  and subscribe to this request as it helps us gauge interest. We will post any updates here.

**Comment by dannyneira** [comment]
Hey Folks, the update released today `v0.2023.03.07.08.02.stable_01` adds this feature to Warp. Closing as resolved. 🚀
